enhancements/metadata/user-tag-management.md

```diff
@@ -0,0 +1,372 @@
+# Metadata Controller
+
+The metadata controller provides capability to add, update, delete and reconcile metadata required for cloud resources.
+The scope of the present proposal is to enable continuous management of user-defined tags for cloud resources. The APIs in the
+proposal have been deliberately designed to be modular for extensibility in the early versions. The API will be refined and updated
+based on use-cases towards beta versions.
+
+## Summary
+
+Tagging cloud resources enable users to perform administrative operations like, organize the resources, 
+apply security policies, optimize operations, etc. The cloud resources can be tagged using cloud service provider tools, 
+kubernetes controllers that create resources via cloud service provider api and other open source tools. A reconciliation 
+using a generic metadata controller helps to keep the tags synchronized for cloud resources. 
+
+In the following proposal, the controller supports mainly 
+1. Day 2 operations and tag management for cloud resources selected using classifiers.
```

Comment:
who applies the classifier tags?  The operator creating the resource?  the admin?  other?

Reply:
> The classifier tag is added by admin. It can be added by installer or operators creating the resource (existing flow of adding tags on cloud resource creation) or using external tool on the cloud resource.

so the admin would provide the classifier tag as an additional tag to be applied, to the installer.

the installer/operators would apply that tag during resource creation

then on day 2 the admin would be able to leverage this controller to define additional tags to be applied to the resources that got the classifier tag?

something about that flow feels awkward ("i want to tag resources, so first i have to tag resources")